editoast/src/client/postgres_config.rs

```diff
@@ -32,4 +33,8 @@ impl PostgresConfig {
             self.psql_database
         )
     }
+
+    pub fn get_connection(&self) -> PgConnection {
```

Comment:
that's a sneaky name, it implies the connection is already established. `make_connection` would be better suited